service-front/module/Application/src/Form/View/Helper/FormRadio.php

```diff
@@ -202,4 +202,9 @@ protected function renderOptions(
 
         return implode($this->getSeparator(), $combinedMarkup);
     }
+
+    public function setViewForTesting()
+    {
+        $this->view = "Set To Something Non-Null To Allow Unit Testing";
```

Comment:
we probably want to revisit this - and raise a ticket to make this better, personally not a fan of magic strings etc.

Reply:
I agree, this is not very pretty :)